windows-support/Makefile

```diff
@@ -12,37 +15,45 @@ wine: /usr/bin/wine
 vendor:
 	mkdir -p vendor
 
+state:
+	mkdir -p state
+
 vendor/python.msi: | vendor state/wineprefix
 	wget https://www.python.org/ftp/python/2.7.10/python-2.7.10.msi -O vendor/python.msi
 	touch vendor/python.msi
 
 vendor/python.msi.installed: | vendor/python.msi
 	cp vendor/python.msi vendor/python-tmp.msi
-	WINEDEBUG=-all WINEPREFIX=$$PWD/state/wineprefix WINEARCH=win32 wine msiexec /i vendor/python-tmp.msi
+	$(WINEARGS) $(WINE) msiexec /i vendor/python-tmp.msi /qn
 	touch vendor/python.msi.installed
 
 state/wineprefix/drive_c/Python27/Scripts/pyinstaller.exe: | vendor/python.msi.installed vendor/pywin32.exe.installed
-	WINEDEBUG=-all WINEPREFIX=$$PWD/state/wineprefix WINEARCH=win32 wine 'c:/python27/python.exe' -m pip install pyinstaller
+	$(WINEARGS) $(WINE) 'c:/python27/python.exe' -m pip install pyinstaller
 
 vendor/pywin32.exe: | vendor
 	wget http://iweb.dl.sourceforge.net/project/pywin32/pywin32/Build%20219/pywin32-219.win32-py2.7.exe -O vendor/pywin32.exe
```

Comment:
If we want to get fancy, at some point we can check hashes of the files we're downloading, similar to https://anonscm.debian.org/cgit/collab-maint/alpine.git/tree/debian/rules#n56 . Not required for this pull request, just wanted to put it on our collective radar.